repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/ClientAuthenticationHandlerFactory.java

```diff
@@ -77,7 +78,12 @@ public void configurationUpdated(ClientAuthConfig modifiedConfig) {
                     accountRegexExtractor.addPattern(clientMapping.getIdRegex());
                 }
                 if (modifiedConfig.getAtomFeeds() != null) {
-                    activateOpenstackAtomFeedListener(modifiedConfig);
+                    try {
+                        activateOpenstackAtomFeedListener(modifiedConfig);
+                    } catch (TimeoutException e) {
```

Comment:
Should this just get swallowed here? What was happening before with the potential NPE? or was there no potential before?